node/protocols/data_query/data_query_test.go

```diff
@@ -89,6 +89,9 @@ func TestDataQueryProtocol(t *testing.T) {
 	assert.True(t, ok)
 	assert.NotEmpty(t, msgs)
 
+	err = protocol2.PurgeQueryHistory(hexutil.Encode(hashOfReq))
```

Comment:
Add more test to check if the data were deleted from the local map

Reply:
@filefilegoadmin done